### PR TITLE
Mobile Menu Interactions

### PR DESCRIPTION
**Pull request by adarachel**
### Implement the following features:

- When the user clicks (or taps) the hamburger button, the mobile menu appears.
- When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @adarachel, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
You have submitted a pair programming project, that does not have [commit from your partner](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#work-and-submission-mode) 


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by adarachel** [comment]
@prolajumokeoni Excuse me, my partner made a commit into this repository already. in fact just a few minutes before I requested a review. I can see her commit on my github commit history, so please why can't you? I am awaiting your response, thank you.

**Comment by prolajumokeoni** [comment]
> @prolajumokeoni Excuse me, my partner made a commit into this repository already. in fact just a few minutes before I requested a review. I can see her commit on my github commit history, so please why can't you? I am awaiting your response, thank you.

- [ ] Hi @adarachel there is currently just one commit on this pr kindly add your commit, both partners should have balanced commits as this is a pair programming project 👍🏿

File: styles.css [hunk 1]
```diff
@@ -30,7 +30,7 @@ button:hover {
 }
 
 button:active {
-  background: #008552;
+  background: #9acd32;
```
Comment (youmari) [inline styles.css:33]: - [ ] the menu works as expected but the close button is bigger than the one in the Figma design so please try to stick to the design and make it the same size Figma design one 
- Yours 
Screen Shot 2022-10-18 at 12 30 51

- Figma
Screen Shot 2022-10-18 at 12 33 54

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @adarachel , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Santiago220991 — APPROVED** [review]
Hi @adarachel @Tessyrich , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
:heavy_check_mark: No linter errors.
:heavy_check_mark: Applied changes suggested by the last reviewer.
:heavy_check_mark: Good PR.

## Optional suggestions
[OPTIOINAL] Kindly make sure that your navbar links are working and redirect the user to that specific section of your portfolio. :pray: :pray: Recruiters definitely are going to test these links.
image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**